client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -36,6 +37,16 @@ class BusinessDetails extends Component {
 		const { addNotice, goToNextStep, isError, updateProfileItems } = this.props;
 		const { other_platform, product_count, selling_venues } = this.state;
 
+		const alreadySelling = [ 'other', 'brick-mortar-other', 'brick-mortar' ].includes(
+			selling_venues
+		);
+
+		recordEvent( 'storeprofiler_store_business_details', {
```

Comment:
Nitpicking, but should these all be prefixed with `wc_store_profiler_` or `wc_admin_store_profiler_`?

Reply:
`wcadmin_` gets appended by the library, so these will actually become `wcadmin_storeprofiler_store_business_details` in tracks.

I think `storeprofiler_` works (and matches the spreadsheet) -- but let me know if you think we should update these.

Reply:
Looks like you're right.  I was confused since this line prefixes the event - https://github.com/woocommerce/woocommerce-admin/blob/81e76bfeb7be39d5a4fa6ac9200db27b53a1c1dd/client/header/activity-panel/index.js#L53

I'll log an issue to handle that.

>I think storeprofiler_ works (and matches the spreadsheet) -- but let me know if you think we should update these.